File: api/filing/models/house_property_address_model.py
from .api_base_model import ApiBaseModel


class HousePropertyAddressModel(ApiBaseModel):
    property_address_id: int | None = None
    filing_id: int
    property_id: int | None = None
    address_line1: str
    address_line2: str | None = None
    city: str
    district: str
    state: str
    postal_code: str
    country: str = "India"
# HousePropertyAddressModel

File: web/filing/models/income/house-property/house-property-address-model.ts
export interface HousePropertyAddressModel {
    propertyAddressId?: number | null
    filingId: number | null
    propertyId?: number | null
    addressLine1: string
    addressLine2?: string | null
    city: string
    district: string
    state: string
    postalCode: string
    country: string
  }